src/components/CanvasWrapper.tsx

```diff
@@ -133,17 +133,20 @@ export default function CanvasWrapper(inputProps: CanvasWrapperProps): ReactElem
   const setBackdropVisible = useViewerStateStore((state) => state.setBackdropVisible);
   const setOpenTab = useViewerStateStore((state) => state.setOpenTab);
   const setTrack = useViewerStateStore((state) => state.setTrack);
-  const showHeaderDuringExport = useViewerStateStore((state) => state.showHeaderDuringExport);
-  const showLegendDuringExport = useViewerStateStore((state) => state.showLegendDuringExport);
-  const showScaleBar = useViewerStateStore((state) => state.showScaleBar);
-  const showTimestamp = useViewerStateStore((state) => state.showTimestamp);
+  // const showHeaderDuringExport = useViewerStateStore((state) => state.showHeaderDuringExport);
+  // const showLegendDuringExport = useViewerStateStore((state) => state.showLegendDuringExport);
+  // const showScaleBar = useViewerStateStore((state) => state.showScaleBar);
+  // const showTimestamp = useViewerStateStore((state) => state.showTimestamp);
```

Comment:
maybe I am jumping the gun, but these could all be `(state.showWhatever && !dataset.is3D)` and then you'd be able to implement the conditional logic and still hardcode the dataset to be 3d-only...

Reply:
What I'm thinking about doing is having the `CanvasOverlay` fully wrap the `Colorize3D`, and it would still take in all of these values. But there are issues with different rendering patterns (on-demand vs. `requestAnimationFrame`) that make me hesitate to integrate the two of them yet.

Reply:
This is fixed in #622!